restapi/views.py

```diff
@@ -48,11 +50,24 @@ class UserViewSet(mixins.UpdateModelMixin,
     permission_classes = [permissions.IsAuthenticated, IsOwnerOrReadOnly]
 
     def update(self, request, *args, **kwargs):
+        user = CustomUser.objects.get(id=kwargs['pk'])
+        old_user_email = user.email
         response = super().update(request, *args, **kwargs)
+        new_user_email = response.data['email']
+
         if response.status_code != 200:
             return response
-        user = CustomUser.objects.get(id=kwargs['pk'])
+        if old_user_email == new_user_email:
+            data = {
+                "detail": "Not_modified."
+            }
+            return Response(data=data, status=status.HTTP_304_NOT_MODIFIED)
```

Comment:
So, basically when there is no email change, but there are some other changes to user fields, we are updating those, but return the not successful response code?